allure-report-face/src/main/webapp/js/testcase/testcase.js

```diff
@@ -91,11 +93,12 @@ angular.module('allure.core.testcase.controllers', ['d3'])
             $scope.setAttachment((direction < 0 ? allAttachments.getPrevious(index) : allAttachments.getNext(index)).uid);
         };
 
-        $scope.getIconClass = function(type) {
+        $scope.getIconClass = function(type) { //NOSONAR
```

Comment:
This block looks perfect. Don't know why sonar is complaining.
What's the warning was here? We can change config on sonar side according our codestyle